tests/unit/s2n_client_key_share_extension_test.c

```diff
@@ -447,8 +447,8 @@ int main(int argc, char **argv)
 
             EXPECT_SUCCESS(s2n_client_key_share_extension.recv(server_conn, &key_share_extension));
 
-            /* should not have initialized any other curves */
-            for (int i = 1; i < ecc_pref->count; i++) {
+            /* should not have initialized any curves */
+            for (size_t i = 0; i < ecc_pref->count; i++) {
```

Comment:
I'm not sure you should swap this over to a size_t if you're otherwise not touching this file.

Reply:
This should have been `0` and not `1`, since I was anyways touching that line, I swapped it to a size_t.